src/libs/Network/index.js

```diff
@@ -236,27 +246,41 @@ function processNetworkRequestQueue() {
         processRequest(queuedRequest)
             .then(response => onResponse(queuedRequest, response))
             .catch((error) => {
+                // Cancelled requests should not be retried
+                if (error.name === CONST.ERROR.REQUEST_CANCELLED) {
+                    onError(queuedRequest, error);
```

Comment:
Nah this log here is enough for me...

https://github.com/Expensify/App/blob/4d646e79cc006aefe4c530b19cb085e1107c9042/src/libs/API.js#L187-L190

One could deduce that this was from the main queue because this log line would not happen

https://github.com/Expensify/App/blob/1bd755465dbb6cbff04a12ba54acd0da5488b4dd/src/libs/Network/index.js#L84-L85

Nobody looks at these logs anyway 😫